### PR TITLE
WebView2: Added release notes for SDK 0.9.579

### PR DESCRIPTION
**Pull request by peiche-jessica**

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -20,6 +20,32 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+## 0.9.579  
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+
+#### General
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/releasenotes.md:27]:
```suggestion
[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

#### General  
```

File: microsoft-edge/webview2/releasenotes.md [hunk 2]
```diff
@@ -20,6 +20,32 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+#### .NET  
+
+*   Improved WPF focus handling.  \([\#185][GithubMicrosoftedgeWebviewfeedbackIssue185]\).
+
+  
+*   Added ZoomFactor property on WPF Webview2 Controller.  
+
+
 ## 0.9.538
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/releasenotes.md:49]:
```suggestion
*   Improved WPF focus handling.  \([\#185][GithubMicrosoftedgeWebviewfeedbackIssue185]\).

*   Added ZoomFactor property on WPF Webview2 Controller.  

## 0.9.538  
```

File: microsoft-edge/webview2/releasenotes.md [hunk 3]
```diff
@@ -20,6 +20,32 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**:  Support for WebView2 SDK Versions [0.8.190](#08190), [0.8.230](#08230), [0.8.270](#08270), [0.8.314](#08314), and [0.8.355](#08355) will be dropped after the next SDK release. These versions are also marked deprecated on nuget.org. The WebView team recommends staying up to date with the latest version of WebView2.
+
+*   Added ICoreWebView2EnvironmentOptions::IsSingleSignOnUsingOSPrimaryAccountEnabled to enable conditional access for WebView.
+*   Added WebResourceResponseReceived event API for authentication request and include authentication headers.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:33]: Recommend something like `Added WebResourceResponseReceived event that is fired when WebView2 receives HTTP responses. Authentication headers, if any, are included in the response object.`

Also, should link to the API/event in reference doc.

File: microsoft-edge/webview2/releasenotes.md [hunk 4]
```diff
@@ -20,6 +20,32 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+*   Added WebView worker thread improvements.  \([\#318][GithubMicrosoftedgeWebviewfeedbackIssue318]\).
+
+*   Disabled popup blocker in WebView, see [ICoreWebView2NewWindowRequestedEventArgs::get_IsUserInitiated](reference/win32/0-9-538/icorewebview2newwindowrequestedeventargs.md#get_isuserinitiated) for more details.
+
+*  Updated WebMessageReceived event to handle null web messages.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:39]: What does this mean? We will fire event for null messages?

**Comment by peiche-jessica** [comment]
@liminzhu Do we need to mention WebView2 RT preview too?

**Comment by liminzhu** [comment]
> @liminzhu Do we need to mention WebView2 RT preview too?

Yes, great callout, thanks for brining it up! We should announce the preview here too, and direct folks to https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#understand-the-webview2-runtime-and-installer-preview for more details.

File: microsoft-edge/webview2/releasenotes.md [hunk 5]
```diff
@@ -20,13 +20,36 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+## 0.9.579  
+Release Date: 7/20/2020
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:24]:
```suggestion
Release Date: 07/20/2020
```

File: microsoft-edge/webview2/releasenotes.md [hunk 6]
```diff
@@ -20,13 +20,36 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: Evergreen WebView2 Runtime and installer is released for preview. See [Distribution of WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#understand-the-webview2-runtime-and-installer-preview) for more details.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:31]:
```suggestion
    > **Announcement**: Evergreen WebView2 Runtime and installer is released for preview. See [Distribution of WebView2](./concepts/distribution#understand-the-webview2-runtime-and-installer-preview) for more details. 
```

File: microsoft-edge/webview2/releasenotes.md [hunk 7]
```diff
@@ -20,13 +20,36 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+* Added experimental [IsSingleSignOnUsingOSPrimaryAccountEnabled][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
+*   Added experimental [WebResourceResponseReceived][ReferenceWin3209538Icorewebview2experimentalAddWebresourceresponsereceived] event that fires after the WebView has received and processed the response for a WebResource request. Authentication headers, if any, are included in the response object.
+*   Added WebView worker thread improvements.  \([\#318][GithubMicrosoftedgeWebviewfeedbackIssue318]\).
+*   Disabled popup blocker in WebView, see [IsUserInitiated][ReferenceWin3209538Icorewebview2newwindowrequestedeventargsGetIsuserinitiated] property in NewWindowRequested event for more details.
+*   Updated WebMessageReceived event to handle null web messages.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:38]: Per offline discussion, we agreed to remove this from the release notes and @liminzhu will inform the relevant team.
```suggestion
```

File: microsoft-edge/webview2/releasenotes.md [hunk 8]
```diff
@@ -20,13 +20,36 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+* Ensured WebView navigation starting event is fired for about:blank. Now, NavigationStarting events are fired for all navigations, but cancellations for about:blank or iframe srcdoc is not supported and ignored.
+* Blocked edge:// URI scheme in WebView.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:40]:
```suggestion
*   Blocked `edge:// URI` scheme in WebView.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 9]
```diff
@@ -20,13 +20,36 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+* Added experimental [IsSingleSignOnUsingOSPrimaryAccountEnabled][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
+*   Added experimental [WebResourceResponseReceived][ReferenceWin3209538Icorewebview2experimentalAddWebresourceresponsereceived] event that fires after the WebView has received and processed the response for a WebResource request. Authentication headers, if any, are included in the response object.
+*   Added WebView worker thread improvements.  \([\#318][GithubMicrosoftedgeWebviewfeedbackIssue318]\).
+*   Disabled popup blocker in WebView, see [IsUserInitiated][ReferenceWin3209538Icorewebview2newwindowrequestedeventargsGetIsuserinitiated] property in NewWindowRequested event for more details.
+*   Updated WebMessageReceived event to handle null web messages.
+* Ensured WebView navigation starting event is fired for about:blank. Now, NavigationStarting events are fired for all navigations, but cancellations for about:blank or iframe srcdoc is not supported and ignored.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:39]:
```suggestion
*   Ensured WebView navigation starting event is fired for about:blank. Now, NavigationStarting events are fired for all navigations, but cancellations for about:blank or iframe srcdoc is not supported and ignored.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 10]
```diff
@@ -20,13 +20,35 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
 
+## 0.9.579  
+Release Date: 07/20/2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: Evergreen WebView2 Runtime and installer is released for preview. See [Distribution of WebView2](./concepts/distribution#understand-the-webview2-runtime-and-installer-preview) for more details.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:31]: My bad -- we should add the `.md`.
```suggestion
    > **Announcement**: Evergreen WebView2 Runtime and installer is released for preview. See [Distribution of WebView2](./concepts/distribution.md#understand-the-webview2-runtime-and-installer-preview) for more details. 
```

**Pull request merged**